requirements.txt

```diff
@@ -9,6 +9,7 @@ datadog==0.21.0
 flake8==5.0.4
 Flask==2.2.2
 honcho==1.1.0
+python-jose[cryptography]==3.3.0
```

Comment:
Just for my own understanding, you had to add these to the pypi repo as well right?

Reply:
yes